src/main/java/org/sunbird/operationalreports/service/OperationalReportServiceImpl.java

```diff
@@ -367,6 +372,39 @@ private long getParsedDate(String reportExpiryDate) throws ParseException {
         return reportExpiryDateMillis;
     }
 
+
+    public SBApiResponse getFileInfo(String authToken) {
+        SBApiResponse response = ProjectUtil.createDefaultResponse(Constants.GET_FILE_INFO_OPERATIONAL_REPORTS);
+        try {
+            logger.info("Inside the getFileInfo()");
+            String userId = accessTokenValidator.fetchUserIdFromAccessToken(authToken);
+            if (null == userId) {
+                response.setResponseCode(HttpStatus.BAD_REQUEST);
+                response.getParams().setStatus(Constants.FAILED);
+                response.getParams().setErrmsg("User Id does not exist");
```

Comment:
return the response immediately in case of error.

Reply:
returning the response object.